### PR TITLE
sorted the components index

### PR DESCRIPTION
**Pull request by nguasch**
I have sorted the component index alphabetically. 

In addition, I have left more space between the elements to be able to add new components in the future

### CODE REVIEW

**Review by jcger — APPROVED** [review]

File: styles/generics/_docs-index.scss [hunk 1]
```diff
@@ -67,53 +67,48 @@ $sgi-iconsSize:                    '1.6.1.2';
...
+$sgi-pagination:             '2.140';
+$sgi-panels:                 '2.150';
+$sgi-rating-nps:             '2.160';
+$sgi-steps:                  '2.165';
+$sgi-steps-bullet:             '2.165.1.1';
+$sgi-tabs:                   '2.170';
+$sgi-tabsMenu:                 '2.170.1.1';
+$sgi-tabsMenuLink:             '2.170.1.2';
+$sgi-tabsIcons:                '2.170.1.3';
+$sgi-thumbnail:              '2.180';
```
Comment (PablitoGS) [inline styles/generics/_docs-index.scss]: here

Comment (nguasch) [reply styles/generics/_docs-index.scss]: done :)

**Comment by jcger** [comment]
Do you only sort by component name? Because there is stuff like

```
$sgi-button-group-structure:         '2.30.1.1';
$sgi-button-group-justified:         '2.30.1.2';
$sgi-button-group-icon:              '2.30.1.3';
```

which is still not sorted alphabetically.

Maybe you should copy all the lines and use some IDE or online tool which will sort it alphabetically for you

**Review by PablitoGS — APPROVED** [review]

**Comment by nguasch** [comment]
@jcger The idea is to sort (alphabetically) only the component and the structure subsections by _logic_ order (because they subsections aren't displayed on the menu)

**Pull request merged**